src/index.rs

```diff
@@ -499,6 +499,27 @@ impl Index {
     )
   }
 
+  #[cfg(test)]
+  pub fn get_output_script(&self, output: &OutPoint) -> Result<ScriptBuf> {
```

Comment:
Should this return an error? Test functions should just panic and not return errors, since an error can be ignored and a panic cannot.